### PR TITLE
bit 0.10.7

### PR DESCRIPTION
**Pull request by amitgilad3**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by ilovezfs — DISMISSED** [review]
This still is not building from source.

**Comment by ilovezfs** [comment]
@amitgilad3 if you need an example, see the `heroku` formula please.

**Comment by amitgilad3** [comment]
@ilovezfs thanks for the example, will go over it and modify the  formula

**Comment by ilovezfs** [comment]
@amitgilad3 thanks!

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
needs a `node` depends_on

File: Formula/bit.rb [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
   homepage "https://www.bitsrc.io"
-  url "https://bitsrc.jfrog.io/bitsrc/bit-brew/stable/bit/0.10.3/bit-0.10.3-brew.tar.gz"
-  sha256 "e843e29b34ba8c93c68503f288be28c841c159b67a54bcc88063328167a32785"
+  url "https://registry.npmjs.org/bit-bin/-/bit-bin-0.10.7.tgz"
+  sha256 "d033b76974be7103933abe1fc937297d29c19aa1cdabbba8d5032ebd9d4f72f0"
+  head "https://github.com/teambit/bit"
 
   bottle do
     cellar :any_skip_relocation
+    sha256 "afc34fb0172b2eb1a579b4334a084c0df77b7ad6977a11d3641a72dca6211487" => :high_sierra
```
Comment (ilovezfs) [inline Formula/bit.rb]: this shouldn't be modified manually

**Comment by amitgilad3** [comment]
@ilovezfs - i changed to formula to work from source

File: Formula/bit.rb (hunk 1 above)
Comment (amitgilad3) [reply Formula/bit.rb]: @ilovezfs - my mistake thanks for the fast reply, i fixed this

File: Formula/bit.rb [hunk 2]
```diff
@@ -1,23 +1,25 @@
 require "language/node"
+
 class Bit < Formula
   desc "Distributed Code Component Manager"
   homepage "https://www.bitsrc.io"
-  url "https://bitsrc.jfrog.io/bitsrc/bit-brew/stable/bit/0.10.3/bit-0.10.3-brew.tar.gz"
-  sha256 "e843e29b34ba8c93c68503f288be28c841c159b67a54bcc88063328167a32785"
+  url "https://registry.npmjs.org/bit-bin/-/bit-bin-0.10.7.tgz"
+  sha256 "d033b76974be7103933abe1fc937297d29c19aa1cdabbba8d5032ebd9d4f72f0"
+  head "https://github.com/teambit/bit"
```
Comment (ilovezfs) [inline Formula/bit.rb]: need .git

File: Formula/bit.rb [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "afc34fb0172b2eb1a579b4334a084c0df77b7ad6977a11d3641a72dca6211487" => :high_sierra
+    sha256 "67a6a52d87f99c90171b82f1f10c784a686969a30366e5c119d6b450a04271bf" => :high_sierra
```
Comment (ilovezfs) [inline Formula/bit.rb]: you shouldn't manually modify the bottle block at all

**Comment by ilovezfs** [comment]
@amitgilad3 there are a couple little changes needed that I've mentioned above. Also, please squash this down to one commit with the commit title `bit 0.10.7` and the commit body `build from source`.

File: Formula/bit.rb (hunk 3 above)
Comment (amitgilad3) [reply Formula/bit.rb]: ok, i wont touch the bottle block in the future

**Comment by ilovezfs** [comment]
@amitgilad3 why the close?

**Comment by amitgilad3** [comment]
beacuse of all the commits and mistakes i will just open a new pull request with all the fixes and correct pull request

**Pull request closed without merging**